documentation/src/docs/asciidoc/release-notes/release-notes-5.3.0-M1.adoc

```diff
@@ -29,6 +29,8 @@ on GitHub.
 
 * New overloaded variant of `isAnnotated()` in `AnnotationSupport` that accepts
   `Optional<? extends AnnotatedElement>` instead of `AnnotatedElement`.
+* Added `--fail-if-no-tests` command line option to `ConsoleLauncher`. When this option is enabled
+and no tests are found for execution the launcher will fail and exit with the non zero code `2`.
 
```

Comment:
For future reference, please note that we wrap at 90 characters, and subsequent lines in a multi-line bullet point entry are indented.

Feel free to change it in the PR. If you don't, we'll do it while/after we merge.

Reply:
Sure I will change this

Reply:
Thanks

Reply:
FYI: I think I might just go ahead and merge this into `master`.

So don't worry about it. 😉